src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/PipsPagerTests/PipsPagerPage.xaml.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Windows.UI.Xaml.Controls;
+using System.Collections.Generic;
+using System.Linq;
+using PipsPagerButtonVisibility = Microsoft.UI.Xaml.Controls.PipsPagerButtonVisibility;
+using PipsPagerSelectedIndexChangedEventArgs = Microsoft.UI.Xaml.Controls.PipsPagerSelectedIndexChangedEventArgs;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Uno.UI.Samples.Controls;
+
+namespace MUXControlsTestApp
+{
+	[Sample("PipsPager", "WinUI")]
```

Comment:
I think we should minimize the need to create a new category for each controls... it's already overcrowded in the sample apps ;-)

Reply:
True :-D . Maybe we could remove the category for the WinUI controls which have just one sample page. Some have more - like the TreeView or NavView, so there the category helps. Or we could introduce hierarchical navigation 😎 !